wandb/apis/public.py

```diff
@@ -662,7 +662,7 @@ def _parse_path(self, path):
 
     def _parse_artifact_path(self, path):
         """Return project, entity and artifact name for project specified by path."""
-        project = self.settings["project"]
+        project = self.settings["project"] or "uncategorized"
```

Comment:
I think using `get` here would be cleaner
```suggestion
        project = self.settings.get("project", "uncategorized")
```
(I would also change the line below to match)